src/Form/EdgeEntityDisplaySettingsForm.php

```diff
@@ -0,0 +1,223 @@
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
+   * @param \Drupal\Core\Entity\EntityDisplayRepositoryInterface $entity_display_repository
+   *   The entity display repository.
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
+   *   The module handler.
+   * @param \Drupal\Core\Routing\RouteMatchInterface $route_match
+   *   The route match.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory, EntityTypeManagerInterface $entity_type_manager, EntityDisplayRepositoryInterface $entity_display_repository, ModuleHandlerInterface $module_handler, RouteMatchInterface $route_match) {
+    parent::__construct($config_factory);
+    $this->entityTypeManager = $entity_type_manager;
+    $this->moduleHandler = $module_handler;
+    $this->entityDisplayRepository = $entity_display_repository;
+    $this->routeMatch = $route_match;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('config.factory'),
+      $container->get('entity_type.manager'),
+      $container->get('entity_display.repository'),
+      $container->get('module_handler'),
+      $container->get('current_route_match')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getBaseFormId() {
+    return 'apigee_edge_display_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    $entity_type_id = $this->routeMatch->getParameter('entity_type_id');
+    return "apigee_edge_display_settings_form.{$entity_type_id}";
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'apigee_edge.display_settings',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state, $entity_type_id = NULL) {
+    $this->entityTypeId = $entity_type_id;
+
+    /** @var \Drupal\Core\Entity\EntityTypeInterface $entity_type */
+    $entity_type = $this->entityTypeManager->getDefinition($entity_type_id);
+    $config = $this->configFactory()->get("apigee_edge.display_settings.{$entity_type_id}");
+
+    $form['display_settings'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Configure the display for @label listing page.', [
+        '@label' => $entity_type->getPluralLabel(),
+      ]),
+      '#collapsible' => FALSE,
+    ];
+
+    $form['display_settings']['display_type'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Type'),
+      '#default_value' => $config->get('display_type'),
+      '#required' => TRUE,
+      '#description' => $this->t('Select <em>Display mode</em> to configure a custom display.'),
+      '#options' => [
+        EdgeEntityListBuilder::DEFAULT_DISPLAY_TYPE => $this->t('Default'),
+        EdgeEntityListBuilder::VIEW_MODE_DISPLAY_TYPE => $this->t('Display mode'),
+      ],
+    ];
```

Comment:
Should we specify the "Default" is a table listing of entities, with links to entity operations?